docs/install-upgrade/upgrade.md

```diff
@@ -5,30 +5,33 @@
 Starting with Beta-1 release and onwards, the upgrade process is more streamlined and fully automated. The control node
 is upgraded in place and the agents/switches is upgraded using the control node.
 
-In order to apply the upgrade, the following steps need to be followed:
+In order to apply the upgrade, use the following instructions:
 
-- use the `hhfab` directory from the initial deployment or init the new one using the configs from the running installation
-- run `hhfab build --mode=manual` to generate fully self-contained (airgap) upgrade package
-  - for control node named `control-1` it will be `result/control-1-install.tgz`
-- upload it to the control node (e.g. using `scp`)
-- unpack and run `hhfab-recipe control upgrade` from the resulting directory
+1. use the `hhfab` directory from the initial deployment or init the new one
+  using the configs from the running installation, [see
+below](#init-hhfab-dir-from-the-running-installation)
+1. run `hhfab build --mode=manual` to generate fully self-contained (airgap)
+   upgrade package for control node named `control-1` it will be `result/control-1-install.tgz`
+1. upload it to the control node (e.g. using `scp`)
+1. unpack and run `hhfab-recipe upgrade control` from the resulting directory
 
 ```bash
-tar xzf control-1-install.tgz
-cd control-1-install
-./hhfab-recipe control upgrade
+tar xzf control-1-install.tgz;
+cd control-1-install;
+sudo ./hhfab-recipe upgrade control;
 ```
 
-It'll do all necessary steps to upgrade the control node and the agents/switches. Resulting version could be checked
-using `kubectl -n fab get fab/default -o=jsonpath='{.status.versions.fabricator.controller}'` and compare to the
-fabricator version in the release notes.
+It'll do all necessary steps to upgrade the control node and the
+agents/switches. The upgrade will cause the control node to **reboot**. The
```

Comment:
It'll not reboot the node, but ask if you want to. We should recommend to do so to apply the OS upgrade.

Reply:
What do you think of the change?